tests/test_meta_cat.py

```diff
@@ -44,6 +45,33 @@ def test_save_load(self):
 
         self.assertEqual(f1, n_f1)
 
+    def test_predict_spangroup(self):
+        Span.set_extension('id', default=0, force=True)
+        Span.set_extension('meta_anns', default=None, force=True)
+
+
+        json_path = os.path.join(os.path.dirname(os.path.realpath(__file__)), 'resources', 'mct_export_for_meta_cat_test.json')
+        self.meta_cat.train(json_path, save_dir_path=self.tmp_dir)
+        self.meta_cat.save(self.tmp_dir)
+        n_meta_cat = MetaCAT.load(self.tmp_dir)
+        assert n_meta_cat.config.general.span_group is None 
+
+        spangroup_name = 'predict_spangroup'
+        n_meta_cat.config.general.span_group = spangroup_name
+        nlp = spacy.blank("en")
+        doc = nlp("No history of diabetes.")
+        span = doc.char_span(14, 22, label="foo_spantype")
+        assert span.text == 'diabetes'
+        doc.spans[spangroup_name] = [span]
+        doc = n_meta_cat(doc)
+
+        # set back to None
+        n_meta_cat.config.general.span_group = None
+        assert doc.spans[spangroup_name][0]._.meta_anns['Status']['value'] == 'Affirmed'
```

Comment:
It is not fully clear to me what is being tested here exactly.

Perhaps it would be clearer if we did these things one part at a time?
I.e first we get the spans (`doc.spans[spangroup_name]`)
Then we check its length to make sure it's 1 (I assume that's what we'd expect - correct me if I'm wrong).
Then we'd get the specific met annotation and make sure its value is correct.